### ISSUE TITLE
Commitlint plugins are ignored when linting is done from `run.js`

### ISSUE THREAD

**Issue by jotanavarro** [wagoid/commitlint-github-action#17]
Rules provided by _commitlint_ plugins are ignored.

For example, when having the following configuration (`commitlint.config.js`):
```
module.exports = {
    plugins: ['commitlint-plugin-jira-rules'],
    extends: ['jira'],
}
```
The rules provided by `commitlint-plugin-jira-rules` are not present and the linting fails due to invalid rules being referenced:
```
RangeError: Found invalid rule names: jira-task-id-empty, jira-task-id-min-length, jira-task-id-max-length, jira-task-id-separator, jira-task-id-case, jira-commit-status-case. Supported rule names are: body-case, body-empty, body-leading-blank, body-max-length, body-max-line-length, body-min-length, footer-empty, footer-leading-blank, footer-max-length, footer-max-line-length, footer-min-length, header-case, header-full-stop, header-max-length, header-min-length, references-empty, scope-case, scope-empty, scope-enum, scope-max-length, scope-min-length, signed-off-by, subject-case, subject-empty, subject-full-stop, subject-max-length, subject-min-length, type-case, type-empty, type-enum, type-max-length, type-min-length
    at Object.<anonymous> (/node_modules/@commitlint/lint/lib/index.js:76:19)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```
The tool (commitlint) behaves correctly when invoked locally using `npx`.

**Comment by jotanavarro** [comment]
I believe this is caused by this line:
https://github.com/wagoid/commitlint-github-action/blob/439bc550a3f26edaeccefac40b7cb2e2dced6001/run.js#L92
Which may be typo.  According to [commitlint documentation](https://commitlint.js.org/#/reference-api?id=commitlintload) the `load` function expects a single `{}`, which is not the case here.  When I run similar code:
```
const config1 = load({ file: 'commitlint.config.js'})
```
It actually load the plugin's rules.

**Comment by wagoid** [comment]
Hey @jotanavarro, thank you for the issue! This was actually caused by the fact that we only passed the rules to the `lint` function, without the rest of config opts like `plugins`. There was a PR before that fixed this but I didn't include in yesterday's release. I've just released [v1.3.0](https://github.com/wagoid/commitlint-github-action/releases/tag/v1.3.0) with the fix.

Regarding your comment about the doc, yeah I noticed this in the documentation when I implemented and their doc is wrong. I tried following the doc at the time and it didn't work, so I saw what is the correct API of the function in the code. Their first parameter is a seed of the object to mutate, and second parameter is the actual config. [You can see it here](https://github.com/conventional-changelog/commitlint/blob/master/%40commitlint/load/src/index.js#L22). At the time I just fixed on my end, will create a PR now to improve their docs so that other folks don't get misleading information from the it again :rocket:
